src/test/unit/symbol-store.test.js

```diff
@@ -82,6 +82,11 @@ describe('SymbolStore', function() {
   });
 
   it('should persist in DB', async function() {
+    // This test outputs a console.error that muddies up test output. Temporarily
+    // disable the output.
```

Comment:
as discussed, just add a reference to https://github.com/dumbmatter/fakeIndexedDB/issues/22 here and let's merge it.